test/smoke/testApps/OpenTelemetryApiSupport/src/main/java/com/microsoft/ajl/simple/TestController.java

```diff
@@ -12,13 +13,22 @@ public String root() {
         return "OK";
     }
 
-
-    @GetMapping("/test")
-    public String test() {
+    @GetMapping("/test-api")
```

Comment:
nit: for consistency, can we use the regular camel case for @GetMapping?  i searched all the naming for @GetMapping, i don't see this new naming pattern.

Reply:
hyphens are much more common then camelCase for urls, better I think to change the other places